### ISSUE TITLE
missing invader tokens

### ISSUE THREAD

**Issue by Aindolur** [WarEmu/WarBugs#16220]
**Expected behavior and actual behavior:**

I am missing 4 invader tokens. I completed a fortress and there should be 6 in my inventory and there is only two.

**Steps to reproduce the problem:**

**Testing Screenshots/Videos/Evidences (always needed):**

![Aindolur_000](https://user-images.githubusercontent.com/63126956/78404637-12ec5100-75bc-11ea-9d92-a4ba62d06eef.jpg)

**Comment by Danked1986** [comment]
Hello i have the also the same issue when the server restarted i lost 8 invader tokens i had 72 now i have 64. 6 from fortress and 2 from defending 2 zones
![Danked_000](https://user-images.githubusercontent.com/63127693/78406485-633ecf80-760b-11ea-803b-3525fe46597a.jpg)
 in t4
some people who spwned in The Maw when the server came up got thier tokens and got to roll on bags from The Maw if possible can i get my invader tokens please

**Comment by Kaelang** [comment]
All invader / royal crests earned within the previous 12 hours have been rolled back due to an exploit reported. 

See below:

'We're starting up the server in a minute. Managed to only roll back a little bit, and targeted specifically at people that seemed like they had a few too many things in their pockets. As we did do a hard crash on the server, there will be up to 15 minutes of rollback for players in general. Invader and Royal currency has been reset to the prior backup, approximately 12 hours ago. That's the best we can really do!'
